scss/standalone/patterns_icons.scss

```diff
@@ -7,3 +7,100 @@
 
 @import '../patterns_icons';
 @include vf-p-icons;
+
+// these icons are not included in vf-p-icons, so include
```

Comment:
If we have separate file with additional icons `patterns_icons-additional` why do we need to add them here?
Or if we add them here - do we still need separate file with additional icons?

I guess maybe we need only one file (either one)?

Reply:
good point!